doc/modules/mod_event_pusher_rabbit.md

```diff
@@ -171,15 +165,17 @@ up flexibility over perfromance.
 ### Publisher confirms
 
 By default publisher confirmations are disabled. However, one-to-one
-confirmations can be enabled . When a worker sends a message to a RabbitMQ
-server it waits for a confirmation from the server before it starts to process
-next message. This approach allows to introduce backpressure on a RabbitMQ server
-connection cause the server can reject messages when it's overloaded. On the
-other hand it can cause performance degratation.
+confirmations can be enabled (see *RabbitMQ connection setup* in
+[*Outgoing connections*](../advanced-configuration/outgoing-connections.md)
```

Comment:
You can created links like `../advanced-configuration/outgoing-connections.md#id-of-the-html-element` to redirect and scroll to the section. The id of the element you can find in the generated html. Please try `mkdocs serve` to locally start the doc server.